.github/workflows/retry-failed-releases.yml

```diff
@@ -0,0 +1,28 @@
+# Copyright VMware, Inc.
+# SPDX-License-Identifier: APACHE-2.0
+
+name: '[CI/CD] Retry release PRs'
+on:
+  schedule:
+    # Every 2 hours
+    - cron: '0 */2 * * *'
+# Remove all permissions by default
+permissions: {}
+jobs:
+  retry-failed-pr-releases:
+    runs-on: ubuntu-latest
+    permissions:
+      actions: write
+    steps:
+      - name: Retry "CI Pipeline" failed runs in releases PRs
+        env:
+          TEMP_FILE: "${{runner.temp}}/failed_runs.json"
+          ACTIONS_API_ENDPOINT: "${{ github.api_url }}/repos/${{ github.repository }}/actions"
+        run: |
+          # Obtain "CI Pipeline" failed runs and filter those from release PRs with less than 3 attempts
+          curl -H 'Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' -X GET -Lkso ${{ env.TEMP_FILE }} ${{ env.ACTIONS_API_ENDPOINT }}/workflows/35553382/runs?status=failure
+          failed_runs_ids="$(jq '.workflow_runs[] | select((.run_attempt < 3) and (.display_title | contains("Release")) and (.head_commit.author.email=="bitnami-bot@vmware.com")).id' ${{ env.TEMP_FILE }})"
+
+          for run_id in ${failed_runs_ids}; do
```

Comment:
I reckon that the number of executions should be limited to avoid a high load in case of general outage

Reply:
Good idea, I added a hard cap of 15 workflow runs to be retried.